index.html

```diff
@@ -0,0 +1,163 @@
+        <h5 class="color-primary col-3 h5 text-md-center col-md-3 p-md-0 mb-md-0">Watch</h5>
+        <p class="color-thirthary col-7 font-secondary col-md text-md-center">Enjoy watching a demo(trailer) for every
+          product.
+        </p>
+      </div>
+      <div
+        class="main-programs mb-0 d-flex align-items-center row mt-2 d-md-flex col-md flex-md-column align-items-center pt-md-2 me-md-3">
+        <img class="victor col-2 col-md-2 p-md-0" src="content/images/communication-svg.svg" alt="victor">
+        <h5 class="color-primary col-3 h5 col-md-3 p-md-0 text-md-center mb-md-0">Tea-Talk</h5>
+        <p class="color-thirthary col-7 font-secondary col-md p-md-0 text-md-center">Take a glass of tea(cofee) and meet
+          famous actors, take
+          selfie with them and enjoy the sight of burg Khalifa.</p>
+      </div>
+      <div
+        class="main-programs d-flex align-items-center row mt-2 col-md d-md-flex flex-md-column align-items-center pt-md-2 me-md-3">
+        <img class="victor col-2 col-md-2 p-md-0 text-md-center" src="content/images/voting-svg.svg" alt="victor">
+        <h5 class="color-primary col-3 h5 col-md-3 p-md-0 text-md-center mb-md-0">Voting</h5>
+        <p class="color-thirthary font-secondary col-7 col-md p-md-0 text-md-center">Talk with each other Think and make
+          your choise and vote.</p>
+      </div>
+      <div
+        class="main-programs d-flex align-items-center row mt-2 col-md d-md-flex flex-md-column align-items-center pt-md-2 me-md-3">
+        <img class="victor col-2 col-md-2 p-md-0" src="content/images/awards-svg.svg" alt="victor">
+        <h5 class="color-primary col-3 h5 col-md-3 p-md-0 text-md-center mb-md-0">Awardes</h5>
+        <p class="color-thirthary col-7 font-secondary col-md p-md-0 text-md-center">We have gifts, sponsorships and
+          awardes for the best three
+          companies.</p>
+      </div>
+      <div class="w-100 d-flex justify-content-center mt-4 mb-4 d-md-none">
+        <button type="button" class="join-us p-3 w-75 font-secondary">Join What Is Next 2022</button>
+      </div>
+    </div>
+    <a class="d-none d-md-flex">
+      <p class="color-thirthary about-page-header text-center h6 mb-5">See The Whole Program</p>
+    </a>
+  </section>
+  <section class="featured-speakers d-flex align-items-center row m-0 page-padding">
+    <h2 class="color-secondary text-center h5 mt-3 mb-3">Featured Speakers</h2>
+    <div class="header-border mb-3"></div>
+    <div id="speakers">
+    </div>
+    <div class="collapse d-md-flex" id="mores">
+    </div>
+    <a class="btn d-md-none mt-3 mb-3" data-toggle="collapse" href="#mores" role="button" aria-expanded="false"
+      aria-controls="mores">
+      MORE
+    </a>
+  </section>
+  <footer class="d-none d-md-block mt-2">
+    <div class="partners d-flex align-items-center row page-padding m-0">
+      <h2 class="color-thirthary font-secondary text-center h2 mt-3 mb-3">Partner</h2>
```

Comment:
## [OPTIONAL]

- [ ] Kindly make sure your headings have short underlines instead of long ones, as the design is defined.

|  Your design |    The template |
----------------|-------------------
![image](https://user-images.githubusercontent.com/65068771/164845116-1a6b474f-73c6-455f-9802-9702268f8e2f.png) | ![image](https://user-images.githubusercontent.com/65068771/164845043-d19ea6ed-d542-4523-a48d-5f9d4d1c84e6.png)

Reply:
@andersonlebon  Thank you very much.
- Made the headings underline shorter according to the required design.